bloom-housing/bloom
# fix: exposing more closed listings

This PR addresses https://github.com/bloom-housing/bloom/issues/4348


## Description
Creates a new env variable to control the max number of closed listings

## How Can This Be Tested/Reviewed?
- add new env variable 
- head to the public site
- make sure closed listings show
- go in and create more closed listings than your env variable
- it should only show up to the max closed listing env variable value

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
